src/components/ListItems.jsx

```diff
@@ -139,11 +171,12 @@ class ListItems extends React.Component {
                         value={this.state.inputNewItem}
                         onKeyPress={this.handleKeyPress}
                         onChange={this.handleChange}
+                        name="newItem"
                     /> :
                     ''
                 }
-                <BtnAddNewItem handleClick={this.handleClick} />
-
+                <BtnAddNewItem onClick={this.handleClickAdd} />
+                {this.state.btnAddNewItemClicked ? <BtnRemoveNewItem onClick={this.handleClickRemove} /> : ''}
```

Comment:
you should use conditionnal rendering here instead of ternary operator because you render a useless empty string in this case: `{this.state.btnAddNewItemClicked && <BtnRemoveNewItem onClick={this.handleClickRemove} />}`

Reply:
Ok, in this case i can replace ternary operator by logical operator &&